src/components/KTable/KTable.vue

```diff
@@ -982,6 +1140,20 @@ export const defaultSorter = (key: string, previousKey: string, sortOrder: strin
     white-space: nowrap;
   }
 
+  th.resize-hover {
+    // creates a 2px "border" on the right - can't use the border because it will "jump"
+    box-shadow: calc(-1 * var(--kui-border-width-20, $kui-border-width-20)) 0 0 0 var(--kui-color-border-decorative-purple, $kui-color-border-decorative-purple) inset;
```

Comment:
question: I'm guessing this was already discussed, but just to be sure, how does the user know the table columns are resizable without first hovering on the table? There is no icon/indicator on the table headers that hint that the columns are resizable.

Reply:
As discussed in stand up, we will use current implementation with the understanding that design team will revisit during `KTable` reskin.